### PR TITLE
Update to sisu-task-scheduler 1.7 artifact (sisu-components bra...

### PR DESCRIPTION
**Pull request by jdillon**
...nch)

Use new sisu component version of the scheduler & updated gav.

### CODE REVIEW

File: nexus/pom.xml [hunk 1]
```diff
@@ -186,7 +186,7 @@
     <plexus-jetty-testsuite.version>2.1</plexus-jetty-testsuite.version>
     <plexus.restlet.bridge.version>1.22</plexus.restlet.bridge.version>
     <plexus-security.version>2.8.3-SNAPSHOT</plexus-security.version>
-    <plexus-task-scheduler.version>1.6.0</plexus-task-scheduler.version>
+    <sisu-task-scheduler.version>1.7-SNAPSHOT</sisu-task-scheduler.version>
```
Comment (peterlynch) [inline nexus/pom.xml]: don't forget to release early after the initial test cycle

**Comment by peterlynch** [comment]
+1

**Comment by adreghiciu** [comment]
+1

**Comment by jdillon** [comment]
https://builds.sonatype.org/job/nexus-oss-its-feature/457/ lnx=FAILED win=SUCCESS

Failure in...

org.sonatype.nexus.integrationtests.nexus3615.Nexus3615ArtifactInfoSecurityIT.checkViewAccess

Not sure its related.

**Comment by nabcos** [comment]
+1, the test fails intermittently on other branches as well (NEXUS-4949)

**Pull request merged**